content/docs/overview.md

```diff
@@ -2489,7 +2489,21 @@ Parametric polymorphism, commonly referred to as "generics", allow the user to c
 ### Explicit parametric polymorphism
 Explicit parametric polymorphism means that the types of the parameters must be explicitly provided.
 #### Procedures
-To specify that a parameter is "constant", the parameters name must be prefixed with a dollar sign `$`. The following example takes two constant parameters to initialize an array of known length:
+As a reminder, all parameters passed into a function are run-time "constants" by default in the sense that they can't have their value changed using `=` directly. Yet, if instead one wishes to use parameter names mutably, then one must indicate so using `:=` like so:
+
+```odin
+sin_tau :: proc(angle_in_cycles: f64) -> f64 {
+    angle_in_cycles := angle_in_cycles   // Enables treating `angle_in_cycles` mutably!
+    
+    TAU :: 2 * math.PI
+    angle_in_cycles *= TAU
+    return math.sin(angle_in_cycles)
+}
+assert(math.abs(sin_tau(0.25) - 1) <= 0.001)   // sin_tau(0.25) is approximately 1
+assert(math.abs(sin_tau(0.75) - -1) <= 0.001)  // sin_tau(0.75) is approximately -1
+```
+
+However, to specify that a parameter must be a **compile-time** constant, which is not the same thing as a run-time constant and may sometimes be necessary or desirable, the parameter's name must be prefixed with a dollar sign `$`. The following example takes two compile-time constant parameters and then uses them to initialize an array of known length:
```

Comment:
There's no such thing as a run-time constant as all constants are resolved at compile time. There are read-only variables, like e.g.:
```cpp
FILE_DATA :: #load("filename")
file_data := FILE_DATA // read-only global containing file data. It *is* a variable, but trying to write to it will segfault (if you work around the compiler errors that tell you you can't mutate it.)
```